bcipy/simulator/helpers/sampler.py

```diff
@@ -79,3 +87,118 @@ def __default_reshaper(self, eeg_responses: List[np.ndarray]) -> np.ndarray:
 
         # make sure this returns (7, 10, 90) for tab_test_dynamic
         return np.array(channels_eeg)
+
+
+def default_reshaper(eeg_responses: List[np.ndarray]) -> np.ndarray:
+    """Default data reshaper.
+
+    Returns
+    -------
+    ndarray with shape (channel_n, trial_n, sample_n)
+    """
+
+    channels_eeg = [[] for i in range(len(eeg_responses[0]))]
+
+    for _i, trial_channels_eeg in enumerate(eeg_responses):
+        for c_i, channel_eeg in enumerate(trial_channels_eeg):
+            channels_eeg[c_i].append(channel_eeg)
+
+    return np.array(channels_eeg)
+
+
+class InquirySampler(Sampler):
+    """Samples an inquiry of data at a time."""
+
+    def __init__(self, data_engine: RawDataEngine, *args, **kwargs):
```

Comment:
Is it possible to remove the` *args` and `**kwargs` arguments? I'd like the samplers to have pretty explicit APIs if possible.